scrapers/Hegre.yml

```diff
@@ -0,0 +1,96 @@
+name: Hegre
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hegre.com/films
+    scraper: sceneScraper
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - hegre.com
+    scraper: galleryScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - hegre.com
+    scraper: performerScraper
+performerByName:
+  action: scrapeXPath
+  queryURL: https://www.hegre.com/?q={}&types=Model
+  scraper: performerSearch
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@property="og:title"]/@content
+      Details:
+        selector: //div[@class="film-subtitle"]/text()|//div[@class="massage-copy"]//text()[normalize-space(.)]
+        concat: "\n\n"
+      Date:
+        selector: //span[@class="date"]/text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Image: //meta[@property="og:image"]/@content
+      Studio:
+        Name:
+          fixed: Hegre
+      Tags:
+        Name: //div[@class="approved-tags"]/a
+      Performers:
+        Name: //a[@class="record-model"]/@title
```

Comment:
We can add the URL also here eg
```yaml
        URL:
          selector: //a[@class="record-model"]//a/@href
          postProcess:
            - replace:
                - regex: ^
                  with: https://www.hegre.com
```
or also make a common var for the anchor selector as it uses the same with the name.

Adding the URL here makes it easier when you create the performer from the scrape scene/gallery menu as apart from the Name the URL will also be populated and then you just have to use that to scrape using the included performer scraper.